Sources/MapboxMaps/Style/Style.swift

```diff
@@ -721,6 +725,71 @@ public final class Style: StyleProtocol {
         }
     }
 
+    /// Adds an image to be used in the style.
+    ///
+    /// This API can also be used for
+    /// updating an image. If the image id was already added, it gets replaced
+    /// by the new image.
+    ///
+    /// The image can be used in
+    /// [`icon-image`](https://www.mapbox.com/mapbox-gl-js/style-spec/#layout-symbol-icon-image),
+    /// [`fill-pattern`](https://www.mapbox.com/mapbox-gl-js/style-spec/#paint-fill-fill-pattern), and
+    /// [`line-pattern`](https://www.mapbox.com/mapbox-gl-js/style-spec/#paint-line-line-pattern).
```

Comment:
Just to confirm – does Xcode autocomplete/documentation support this format?

Reply:
Seems like it does

<img width="506" alt="Screenshot 2022-04-19 at 15 47 08" src="https://user-images.githubusercontent.com/1478430/164007385-a9d82178-222f-49fb-bb41-e9244902873b.png">